### PR TITLE
Release fixups

### PR DESCRIPTION
**Pull request by doanac**

### CODE REVIEW

**Comment by doanac** [comment]
fixes - https://github.com/foundriesio/fioctl/issues/155

**Comment by mike-scott** [comment]
@doanac Just the one question, but the approach looks good.

File: subcommands/docker/cmd.go [hunk 1]
```diff
@@ -40,6 +41,9 @@ docker-credential-fio, in the same directory as the docker client binary.
 
 NOTE: The credentials will need the "containers:read" scope to work with Docker`,
 		Run: doDockerCreds,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			subcommands.DieNotNil(err, "Docker not found on system")
```
[the remark this answers is not in the document]
Comment (doanac) [reply subcommands/docker/cmd.go:45]: yep. i tested it a couple of times with /usr/bin/docker missing.

**Review by mike-scott — APPROVED** [review]
LGTM

**Pull request merged**